content/technologies/ar/Alfanous_Build_Process.md

```
@@ -0,0 +1,155 @@
+| `src/alfanous/indexes/` | فهارس بحث بأسلوب Lucene |
+
+---
+
+## 6. مثال على تنفيذ البناء
+
+```bash
+make build
+```
+
+ينفذ هذا الأمر سلسلة العمليات التالية:
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)